src/Razor/Microsoft.AspNetCore.Razor.Tools/test/RewriteCssCommandTest.cs

```diff
@@ -251,20 +266,42 @@ public void RewritesAnimationNamesWhenMatchingKnownKeyframes()
         public void RewritesMultipleAnimationNames()
         {
             // Arrange/act
-            var result = RewriteCssCommand.AddScopeToSelectors(@"
+            var result = RewriteCssCommand.AddScopeToSelectors("file.css", @"
     .myclass1 { animation-name: my-animation , different-animation }
     .myclass2 { animation: 4s linear 0s alternate my-animation infinite, different-animation 0s }
     @keyframes my-animation { }
     @keyframes different-animation { }
-", "TestScope");
+", "TestScope", out var diagnostics);
 
             // Assert
+            Assert.Empty(diagnostics);
             Assert.Equal(@"
     .myclass1[TestScope] { animation-name: my-animation-TestScope , different-animation-TestScope }
     .myclass2[TestScope] { animation: 4s linear 0s alternate my-animation-TestScope infinite, different-animation-TestScope 0s }
     @keyframes my-animation-TestScope { }
     @keyframes different-animation-TestScope { }
 ", result);
         }
+
+        [Fact]
+        public void RejectsImportStatements()
+        {
+            // Arrange/act
+            RewriteCssCommand.AddScopeToSelectors("file.css", @"
+    @import ""basic-import.css"";
+    @import ""import-with-media-type.css"" print;
+    @import ""import-with-media-query.css"" screen and (orientation:landscape);
+    @ImPoRt /* comment */ ""scheme://path/to/complex-import"" /* another-comment */ screen;
```

Comment:
Might be valuable to validate that the CSS parser we're using doesn't represent import statements in comments. I normally wouldn't be concerned with it but given we didn't write the parser it'd be valuable to sanity check. Not saying we need a test for it, just a manual validation to ensure our assumptions are correct

Reply:
Added a case to the unit tests